index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css ">
+
+    <style>
+        body{
+        float: center;
+        margin-left :100px;
+        margin-right :100px;
+        }
+    </style>
+
+</head>
+<body>
+
+<h1>Rejestracja</h1>
+<div>
+    <label for="name">Imię</label>
+    <br>
+    <input type="text" id="name" placeholder="Wpisz imię">
+</div>
+<div>
+    <label for="name">Nazwisko</label>
+    <br>
+    <input type="text" id="surname" placeholder="Wpisz nazwisko">
+</div>
+<div>
+    <p>Płeć</p>
+    <label for="woman">Kobieta</label>
+    <input type="radio" name="navi" id="woman">
+    <br>
+    <label for="man">Mężczyzna</label>
+    <input type="radio" name="navi" id="man">
+</div>
+<div>
+    <label for="email">Adres email</label>
+    <br>
+    <input type="email" placeholder="Wpisz adres email" id="email">
+</div>
+<div>
+    <label for="password">Hasło</label>
+    <br>
+    <input type="email" placeholder="hasło" id="password">
+</div>
+<p style="color:grey;">Min. 8 znaków</p>
+
+<div class="place">
```

Comment:
jak można to zrobić by było w jednej linii? Przychodzi mi na myśl tabela. Jeśli tabela to dla każdej kolumny ustawić inny nagłówek by się zmieściły w jednej linii ? Lepiej wtedy zrobić inline czy w <head> w style formatowanie?

Reply:
Do większości elementów powinnaś wykorzystać style z bootstrapa :P Ułatwi to tobie pracę i sprawi, że twoja strona będzie wyglądać podobnie do tej z treści zadania. Natomiast do ustawienia tego w jednej linii powinnaś wykorzystać wiersze i kolumny z bootsstrapa:
https://getbootstrap.com/docs/4.5/layout/grid/#how-it-works
https://www.w3schools.com/bootstrap4/bootstrap_grid_system.asp